### PR TITLE
Python 3: zip function -> list(zip), itertools.izip -> zip()

### PR DESCRIPTION
**Pull request by josephsl**
Hi,

Discovered through a series of failing unit tests:

### Link to issue number:
None

### Summary of the issue:
Python 2 and 3 zip functions have different behaviors: returns a list in Python 2 vs an iterator in Python 3.

### Description of how this pull request fixes the issue:
Changed instances of zip() function call to list(zip), as well as rename itertools.izip to zip().

Steps:

1. Grep "zip(" source.
2. First, wrap language handler zip function inside a list call.
3. Convert itertools.izip to zip and remove itertools import from IA1 mozilla text module.

### Testing performed:
Tested in Python 2 and 3 interpreters.

### Known issues with pull request:
None

### Change log entry:
None

### CODE REVIEW

File: source/NVDAObjects/IAccessible/ia2TextMozilla.py
Comment (leonardder) [inline source/NVDAObjects/IAccessible/ia2TextMozilla.py]: Please split this line while at it

File: source/languageHandler.py
Comment (leonardder) [inline source/languageHandler.py]: I don't think this comment is necessary. Furthermore, it contains a typo.

**Review by LeonarddeR — CHANGES_REQUESTED** [review]

**Comment by josephsl** [comment]
Hi, done – hopefully the split lines are what you are looking for. Thanks.

**Comment by leonardder** [comment]
I ran through all the items of `git grep "\Wstr\W"`, and all seems wel indeed.

**Comment by michaelDCurran** [comment]
@leonardder  are you still waiting for changes on this, or are you going to approve?

**Review by LeonarddeR — APPROVED** [review]
Yes, go ahead.

**Review by michaelDCurran — APPROVED** [review]

**Comment by michaelDCurran** [comment]
After merging this, setup.py failed as it found a syntax error in source\NVDAObjects\IAccessible\ia2TextMozilla.py, introduced by this pr. Due to a for loop being split across two lines.
I have manually fixed this in 5da17fd0, which is now on threshold_py3_staging.
I appreciate that shorter line lengths are easier to read for some people, but I would suggest we back down on this a bit, especially with reviewing code for the Python3 transition. Not only was there this syntax error, but a lot of other code is having to be slightly rewritten, which really has nothing to do with Python3 and could introduce further bugs.

**Comment by leonardder** [comment]
That makes sense. I'll refrain from review comments requesting for line 
splitting.

**Pull request merged**
